src/test/resources/github427/dir with spaces/rs_20160518_014000_udsc64_l3_e_v10.tab

```diff
@@ -0,0 +1,444 @@
+00001  2016-05-18T02:24:21.556  516810329.741615      15307.329725      15297.461237    8410932000.000000      6092.292  -0.9999E+01     90245.596477  -9999999.999999          1.0010332740      1033.274000  -9999999.999999  -0.9999E+01    -99999999.999999      6098.587  -0.9999E+01      22.87      -6.25

```

Comment:
@al-niessner this is an example where we really don't need this test file to be anything more than a few bytes. can we just make this 1-10 lines and update the label (`<records>`, `<file_size>`, etc.) to validate successfully with that?

Reply:
Changing it is more hassle than it is worth. Have to then update XML too for a bunch of trivia. It would take a day to show that the mods made to table and xml are concurrent then fix the problem you care about which is spaces in the dirname. I just stole #690 which had 0 errors and called it good enough.

Reply:
shrunk while waiting on ideas for how to get cucumber to accept spaces